geom/gdml/inc/TGDMLWrite.h

```diff
@@ -228,8 +232,6 @@ class TGDMLWrite : public TObject {
    TString GetPattAxis(Int_t divAxis, const char * pattName, TString& unit);
    Bool_t IsNullParam(Double_t parValue, TString parName, TString objName);
    void UnsetTemporaryBits(TGeoManager * geoMng);
-   UInt_t GetFltPrecision() const { return fFltPrecision; }
-   void SetFltPrecision(UInt_t prec) { fFltPrecision = prec; }
```

Comment:
The accessors were private because the main interface to set the export precision is intended to be TGeoManager::Set/GetExportPrecision. It was done like this because in most cases TGDMLWrite is immaterial since the users just call TGeoManager::Export. Calling public setters in the TGDMLWrite will anyway be overridden by the default value held by TGeoManager, when using Export. What is the sequence of export you use, I think if it is not via TGDMLWrite::StartGDMLWriting but rather directly WriteGDMLFile, we should rather move the `writer->SetFltPrecision` call inside this lower-level method implementation.